app/src/main/java/it/quip/android/fragment/QuipFeedFragment.java

```diff
@@ -0,0 +1,82 @@
+package it.quip.android.fragment;
+
+import android.os.Bundle;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import it.quip.android.R;
+import it.quip.android.adapter.QuipsAdapter;
+import it.quip.android.model.Quip;
+import it.quip.android.partials.EndlessScrollHandler;
+import it.quip.android.util.MockUtils;
+
+public class QuipFeedFragment extends BaseFragment {
+
+    @Override
+    public CharSequence getTitle() {
+        return "Quips";
```

Comment:
I think there's a potential bug here (my fault). If we have multiple fragments inside of an activity or another fragment, there will be a race condition for setting the title since they will all try to set it at `onResume`. I'll think about a better way to do this